app/models/organisation.rb

```diff
@@ -19,4 +21,10 @@ def self.fetch_organisations_from_register
       organisations_gateway: Gateways::GovukOrganisationsRegisterGateway.new
     ).execute.sort
   end
+
+  def generate_uuid
+    return if uuid.present?
+
+    self.uuid = SecureRandom.uuid
+  end
```

Comment:
Can achieve this without the need for ruby I think:
```
class AddUuidToBooks < ActiveRecord::Migration
  def change
    add_column :organisations, :uuid, :uuid, default: 'uuid_generate_v4()'
  end
end
```

Reply:
👍